applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -50,6 +60,8 @@ struct le_audio_headset {
 	struct bt_conn *headset_conn;
 	struct net_buf_pool *iso_tx_pool;
 	atomic_t iso_tx_pool_alloc;
+	struct bt_codec_qos_pref pref_qos;
```

Comment:
The ep already contains pref_qos and it might be different from the two ep. E.g. you could have 10 ms pres delay for the sink and 30 for the source